components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -2198,17 +2195,15 @@ void AdsServiceImpl::RecordP2AEvent(const std::string& name,
 }
 
 void AdsServiceImpl::LogTrainingInstance(
-    const brave_federated::mojom::TrainingInstancePtr training_instance) {
+    brave_federated::mojom::TrainingInstancePtr training_instance) {
   if (!ad_notification_timing_data_store_) {
     return;
   }
 
-  // TODO(https://github.com/brave/brave-browser/issues/21189): Refactor DB to
-  // use generic key/value schema across all data stores
-  brave_federated::AdNotificationTimingTaskLog log;
   auto callback =
       base::BindOnce(&AdsServiceImpl::OnLogTrainingInstance, AsWeakPtr());
-  ad_notification_timing_data_store_->AddLog(log, std::move(callback));
+  ad_notification_timing_data_store_->AddTrainingInstance(
```

Comment:
The function is named `LogTrainingInstance` however, we then AddTrainingInstance. Can you please explain difference between Log and Add for training instances

Reply:
I can consolidate the two if needed, but I thought of it as the ads service logs the training instance by adding it to the data store

Reply:
I just need to understand the difference between adding/logging, maybe best to go over on a call as above